Assets/Scripts/Entity/Player/Possession.cs

```diff
@@ -112,7 +160,7 @@ private void NotifyOnPossessionControllers(GameObject enemy)
             }
         }
         return nearestEnemy;
-
+        
     }
-
+    
```

Comment:
Porque all these whitespaces? They actually trigger my IDE
![image](https://user-images.githubusercontent.com/1427401/54079236-32c12080-42a7-11e9-8563-a2600c1c6a5c.png)

Reply:
lol no idea why that happened